op-guide/offline-ansible-deployment.md

```diff
@@ -0,0 +1,365 @@
+
+    - A remote user account which you can use to login from the Control Machine to connect to the managed nodes via SSH. It can be the root user or a user account with sudo privileges.
+
+    - Python 2.6 or Python 2.7
+
+    > **Note**: The Control Machine can be one of the managed nodes.
+
+## Install Ansible in the Control Machine
+
+1. Install Ansible offline on CentOS:
+
+  > Download the [Ansible](https://download.pingcap.org/ansible-2.3-rpms.el7.tar.gz) offline installation package to the Control Machine.
+  
+  ```ini
+  
+  tar -xzvf ansible-2.3-rpms.el7.tar.gz
+  
+  cd ansible-2.3-rpms.el7
+  
+  rpm -ivh PyYAML*.rpm libtomcrypt*.rpm libtommath*.rpm libyaml*.rpm python-
+  babel*.rpm python-backports*.rpm python-backports-ssl_match_hostname*.rpm
+  python-httplib2*.rpm python-jinja2*.rpm python-keyczar*.rpm python-
+  markupsafe*.rpm python-setuptools*.rpm python-six*.rpm python2-crypto*.rpm
+  python2-ecdsa*.rpm python2-paramiko*.rpm python2-pyasn1*.rpm sshpass*.rpm
+  rpm -ivh ansible-2.3.1.0-1.el7.noarch.rpm
+  ```
+
+2. After Ansible is installed, you can view the version using `ansible --version`.
+  
+  ```
+  ansible --version
+  # ansible 2.3.1.0
+  ```
+
+## Download TiDB Installation Packages
+
+| Component | Download link | 
+| -------- | ----  | 
+|**Deploy**| [ tidb-ansible release-1.0 ](https://github.com/pingcap/tidb-ansible/tree/release-1.0) | 
+|**TiDB**| [ tidb-1.0.0 ](http://download.pingcap.org/tidb-v1.0.0-linux-amd64-unportable.tar.gz) |
+| | [ tidb-tools-latest ](http://download.pingcap.org/tidb-tools-latest-linux-amd64.tar.gz) | 
+| | [ tidb-binlog-latest ](http://download.pingcap.org/tidb-binlog-latest-linux-amd64.tar.gz) |
+|**Monitor**| [ prometheus-1.5.2 ](https://github.com/prometheus/prometheus/releases/download/v1.5.2/prometheus-1.5.2.linux-amd64.tar.gz) | 
+| | [ grafana-4.1.2 ](https://grafanarel.s3.amazonaws.com/builds/grafana-4.1.2-1486989747.linux-x64.tar.gz) |
+| | [ node_exporter-0.14.0-rc.1 ](http://download.pingcap.org/node_exporter-0.14.0-rc.1.linux-amd64.tar.gz) | 
+| | [ pushgateway-0.3.1 ](http://download.pingcap.org/pushgateway-0.3.1.linux-amd64.tar.gz) |
+| | [ alertmanager-0.5.1 ](https://github.com/prometheus/alertmanager/releases/download/v0.5.1/alertmanager-0.5.1.linux-amd64.tar.gz) |
+| | [ daemontools-0.53 ](http://oifici4co.bkt.gdipper.com/daemontools-0.53.tar.gz) |
+|**Test**| [ fio-2.16 ](https://download.pingcap.org/fio-2.16.tar.gz) | 
+|**Spark**| [ spark-2.1.1-bin-hadoop ](http://download.pingcap.org/spark-2.1.1-bin-hadoop2.7.tgz) |
+| | [ tispark-SNAPSHOT-jar-with-dependencies ](http://download.pingcap.org/tispark-SNAPSHOT-jar-with-dependencies.jar) |
+| | [ tispark-sample-data ](http://download.pingcap.org/tispark-sample-data.tar.gz) |
+
+> **Note**: Download all installation packages to the Control Machine.
```

Comment:
Move this node to the beginning of this section. This content should not be on a note. It is rather the introduction of the section.

Reply:
installation packages -> packages